docs/accessControl.md

```diff
@@ -198,6 +199,32 @@ To block students from viewing closed assessment scores, set `"showClosedAssessm
 
 The `showClosedAssessment` access rule restriction is only useful in conjunction with [disabling real-time grading](assessment.md#disabling-real-time-grading) and setting `"showClosedAssessment": false`.
 
+## Submittable assessments
+
+The `submittable` access rule restriction is useful for allowing students to see what assessments they have coming up. If `submittable` is set to `false`, students can see the assessment on the Assessments page, but they cannot create a new assessment instance or submit answers to questions. If an assessment is currently not submittable but will be in the future, students can see when the assessment will become submittable by looking at the `Available credit` column on the Assessments page. Assessments are submittable by default.
+
+**Note**: if `submittable` is set to `false` in an access rule, the available `credit` for that access rule must be 0.
```

Comment:
```suggestion
**Note**: if `submittable` is set to `false` in an access rule, the available `credit` cannot be set to any value other than 0 (the default value).
```

Just to clarify that they don't need to set the credit, they only need to ensure that it is not set to a non-zero value.